src/modules/gcloud.rs

```diff
@@ -124,7 +124,9 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
                             .map_or(project, Cow::Borrowed)
                     })
                     .map(Ok),
-                "active" => Some(Ok(Cow::Borrowed(&gcloud_context.config_name))),
+                "active" if gcloud_context.config_name != "NONE" => {
+                    Some(Ok(Cow::Borrowed(&gcloud_context.config_name)))
+                }
```

Comment:
I think I misread the original request. I think they wanted the whole module to return `None` if `gcloud_context.config_name` is `NONE`, which would be handled via a regular `if`-condition ahead of the `let parsed` part.:
```
if gcloud_context.config_name == "NONE" {
    return None;
}
```

The documentation for this would need to be in the main description after `# Google Cloud`, like the conditions listed in the AWS module.

Reply:
I'll convert this to a draft and make the adjustment. Thanks! I was thinking maybe there was a little confusion but wasn't sure enough of the ask.

I'm thinking I'd need a more clever test, no? Idk if this tests that the module is *disabled*. Which I'd think is a Module that's None somehow? I can play with it but any advice here would be super appreciated. :)

Reply:
The `ModuleRenderer` will return `None` if the module has no output. I think the test is fine, assuming it fails if `active` is not set to `NONE`.

Reply:
Ready to mark this as resolved if you are. I can give it a double take tomorrow. I think this achieves what we'd like. Would love a gcloud user to validate this PR though. 😇